README.md

```diff
@@ -238,9 +198,31 @@ I would like to thank my coding partners and microverse
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FAQ (optional) -->
-<!-- 
+
 ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
-<!-- 
-> Add at least 2 questions new developers would ask when they decide to use your project. -->
-<!-- 
-- **[Question_1]**l
\ No newline at end of file
+> Add at least 2 questions new developers would ask when they decide to use your project.
+- **[RESOURCES_used_for_writing_this_project]**
```

Comment:
- [ Optional ] This section of the README is meant to guide you on what to write when you try to customize the section. Kindly always omit it when you do your projects. Good job 👏